ui/00_message/common/spot_name.xml

```diff
@@ -1543,10 +1564,12 @@
   <message>
     <key>SPOT_NAME_833</key>
     <original>モルトバ遺跡　左路</original>
+    <translation>Mergova Ruins Left Road</translation>
```

Comment:
I used Mergova since that was used in other translations in the file, so I went ahead and changed all to Moltova.